src/cli/commands/beacon.ts

```diff
@@ -36,11 +41,22 @@ export class BeaconNodeCommand implements CliCommand {
           address: options.depositContract
         },
         provider: await this.getProvider(options)
+      },
+      rpc: {
+        apis: this.setupRPC(options.rpc.split(","))
       }
     });
     await node.start();
   }
 
+  private setupRPC(args: string[]): IApiConstructor[] {
+    return Object.keys(RPCApis).map((api: string) => {
+      if (api.substring(0,1) !== 'I') {
```

Comment:
Interfaces don't exist at runtime so this condition will never be false

Reply:
I tested this out, it pulls them out as strings and is readable. feel free to verify by `console.log(Object.keys(RPCApis))` also make sure you build
`yarn build && ./bin/lodestar beacon --rpc beacon,validator`

Reply:
tbh thouhg, im not even reading the args correctly. Needs a refactor

Reply:
Interesting, the interfaces are exported, but exported as `undefined`... learn something new every day.
You should be able to get a list of Apis then w/ something like:
`Object.values(RPCApis).filter((api) => api !== undefined)`

Reply:
So when running lodestar
```
Successfully compiled 111 files with Babel.
✨  Done in 3.33s.
{ ValidatorApi: [Getter],
  IValidatorApi: [Getter],
  BeaconApi: [Getter],
  IBeaconApi: [Getter] }
```

Reply:
Its a bit crazy lol, So that's why I did the `I` prefix hack, not ideal, but it works

Reply:
And logging the keys:
`console.log(Object.keys(RPCApis)`
```
[ 'ValidatorApi', 'IValidatorApi', 'BeaconApi', 'IBeaconApi' ]
```

Reply:
Try printing the line above, it should be a bit nicer than the I prefix hack.

Reply:
oh neat that works